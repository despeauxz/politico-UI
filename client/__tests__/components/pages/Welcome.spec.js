import React from 'react';
import WelcomeComponent from '@components/pages/Welcome';

const props = {
    user: {},
    isAuthenticated: false,
    authenticating: false
};

describe('Welcome', () => {
    it('should render the Welcome component correctly when unauthenticated', () => {
        const shallowWrapper = shallow(<WelcomeComponent {...props} />);

        expect(toJson(shallowWrapper)).toMatchSnapshot();
        expect(shallowWrapper.find('Welcome')).toBeTruthy();
    });
});