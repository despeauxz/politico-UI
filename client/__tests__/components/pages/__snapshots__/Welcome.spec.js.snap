// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome should render the Welcome component correctly when unauthenticated 1`] = `
<Fragment>
  <div
    className="overlay"
  >
    <h1>
      Vote Right
    </h1>
  </div>
  <Connect(HomeHeader) />
  <div
    className="container"
  >
    <div
      className="page_header"
    >
      <section
        className="intro"
      >
        <h1>
          Make your vote count
        </h1>
        <p
          className="subhead"
        >
          Voting is not our right, it's our power.
        </p>
        <Link
          className="cta_link"
          to="/auth/signup"
        >
          Sign Up
        </Link>
      </section>
      <section>
        <img
          alt="undraw icon"
          src={
            Object {
              "process": [Function],
            }
          }
        />
      </section>
    </div>
  </div>
  <div
    className="blue-container"
  >
    <div
      className="container"
    >
      <h2>
        HOW TO VOTE
      </h2>
      <ul>
        <li>
          <p
            className="text-md"
          >
            1
          </p>
          <p>
            Lorem ipsum dolor sit amet consectetur adipisicing elit. Facere, accusantium ipsam? Alias dolorem fuga eligendi, cum odio recusandae laboriosam
          </p>
        </li>
        <li>
          <p
            className="text-md"
          >
            2
          </p>
          <p>
            Lorem ipsum dolor sit amet consectetur adipisicing elit. Facere, accusantium ipsam? Alias dolorem fuga eligendi, cum odio recusandae laboriosam
          </p>
        </li>
        <li>
          <p
            className="text-md"
          >
            3
          </p>
          <p>
            Lorem ipsum dolor sit amet consectetur adipisicing elit. Facere, accusantium ipsam? Alias dolorem fuga eligendi, cum odio recusandae laboriosam
          </p>
        </li>
      </ul>
    </div>
  </div>
  <div
    className="gray-container"
  >
    <div
      className="container"
    >
      <ul
        className="slider-container"
        id="slider"
      >
        <li
          className="slide"
        >
          <figure>
            <blockquote>
              We do not have government by the majority, We have government by the majority who participate
            </blockquote>
            <figcaption>
              - Thomas Jefferson
            </figcaption>
          </figure>
        </li>
      </ul>
    </div>
  </div>
  <div
    className="container enterprise"
  >
    <h2>
      Get your voting ID now!
    </h2>
    <p
      className="cta"
    >
      Get Started
    </p>
  </div>
  <footer>
    <div
      className="footer-container"
    >
      <div
        className="container"
      >
        <p
          className="desc"
        >
          Designed by Malik Godwin Onimisi
        </p>
        <p>
          Andela Bootcamp Challenge, Cycle 41
        </p>
        <ul
          className="footer-links"
        >
          <li>
            <Link
              to="#"
            >
              Terms of Service
            </Link>
          </li>
          <li>
            <Link
              to="#"
            >
              Privacy Policy
            </Link>
          </li>
        </ul>
      </div>
    </div>
  </footer>
</Fragment>
`;
